packages/consumption/src/modules/attributes/events/AttributeCreatedEvent.ts

```diff
@@ -1,5 +1,5 @@
 import { TransportDataEvent } from "@nmshd/transport";
-import { LocalAttribute } from "../local/LocalAttribute";
+import { LocalAttribute } from "../local/attributeTypes";
```

Comment:
Please check for those imports, I'd prefer exact imports over barrels to avoid cdeps.